src/Application.vala

```diff
@@ -140,18 +140,13 @@ public class Mail.Application : Gtk.Application {
         add_action (quit_action);
         set_accels_for_action ("app.quit", {"<Control>q"});
 
-        /* Needed to ask the flatpak portal for autostart and background permissions with a parent window
-           and to prevent issues with Session.start being called from the InboxMonitor first */
-        if (!run_in_background) {
-            activate ();
-        }
-
         new InboxMonitor ().start.begin ();
         hold ();
```

Comment:
i insist, hold() shouldn't be called in startup(), this will keep the application alive even from a activation without any further action.

Reply:
Oh sorry I thought you were only referring to `request_background ()`